### PR TITLE
quick fix for disabling registries in diff cloud

### PR DESCRIPTION
**Pull request by ianedwards**
## What does this PR do?
Screenshot 2024-01-17 at 10 11 05 AM

### CODE REVIEW

File: dashboard/src/main/home/modals/IntegrationsModal.tsx [hunk 1]
```diff
@@ -18,40 +20,73 @@ export default class IntegrationsModal extends Component<PropsType, StateType> {
...
+          integration.auth_mechanism;
 
         if (!disabled) {
-          return (
-            <IntegrationOption
+          return !sameCloudProvider ? (
+            <Tooltip
               key={i}
-              disabled={disabled}
+              content="Must be on the same cloud provider as your current cluster"
```
Comment (ferozemohideen) [inline dashboard/src/main/home/modals/IntegrationsModal.tsx:69]: maybe "Can only connect registries hosted in the same cloud provider your cluster is provisioned with" if that's not too wordy

**Review by ferozemohideen — APPROVED** [review]

File: dashboard/src/main/home/modals/IntegrationsModal.tsx (hunk 1 above)
Comment (ianedwards) [reply dashboard/src/main/home/modals/IntegrationsModal.tsx]: trimmed it down a bit to fit in the tooltip
